This only includes the sysusers.d portion of
5b5d82615011b9827466b7cd5756da35627a1608 as networkd is not enabled and
avoids patch conflict
---

diff --git a/sysusers.d/systemd.conf.m4 b/sysusers.d/systemd.conf.m4
index 2315b56e3fc..ef5a3cb6198 100644
--- a/sysusers.d/systemd.conf.m4
+++ b/sysusers.d/systemd.conf.m4
@@ -6,6 +6,15 @@
 #  (at your option) any later version.
 
 g systemd-journal   - -
+m4_ifdef(`ENABLE_NETWORKD',
+u systemd-network   - "systemd Network Management"
+)m4_dnl
+m4_ifdef(`ENABLE_RESOLVE',
+u systemd-resolve   - "systemd Resolver"
+)m4_dnl
+m4_ifdef(`ENABLE_TIMESYNCD',
+u systemd-timesync  - "systemd Time Synchronization"
+)m4_dnl
 m4_ifdef(`ENABLE_COREDUMP',
 u systemd-coredump  - "systemd Core Dumper"
 )m4_dnl
